lm_eval/tasks/ko_commongen_v2/code_switching/utils.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+import datasets
+
+
+
+def process_docs(dataset: datasets.Dataset) -> datasets.Dataset:
+    def _process_doc(doc):
+        query = (
+            f"""concept set: {{{doc['concept_set'].replace("#", ", ")}}}\n""")
+        query += "\n".join([f"{i+1}. {doc[str(i+1)]}" for i in range(4)])
+
+        out_doc = {
+            "query": query,
+            "choices": [f"{i+1}. {doc[str(i+1)]}" for i in range(4)],
+            # "choices": [f"{doc[str(i+1)]}" for i in range(4)],
+            # "choices": [f'{str(i+1)}. ' + doc['{i}'.format(i=i + 1)] for i in range(4)],  # The list of choices.
+            # "choices": [str(i+1) for i in range(4)],  # The list of choices.
```

Comment:
Just want to check if this is an alternative option (which is why it's commented but left in)?

Reply:
+1, if this comment is safe to delete let's do so!